src/index.html

```diff
@@ -0,0 +1,1302 @@
+            </li>
+            <li>
+              <a href="#">
+                README
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                GROWTH QUATERS
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                NEURAL
+              </a>
+            </li>
+          </ul>
+          <div class="flex search-aside-buttons">
+            <a href="#">
+              <i class="fas fa-search"></i>
+            </a>
+            <button class="nav-btn" type="button">
+              <i class="fas fa-bars"></i>
+            </button>
+          </div>
+        </div>
+
+      </div>
+
+    </nav>
+    <!-- START of header section -->
+
+    <header>
+
+      <div class="header-articles">
+        <div class="left-box">
+          <img src="../src/assets/svg/TNW-white.svg" alt="main-logo">
+          <p class="date">
+            Wednesday - July 29, 2020
+          </p>
+          <a class="head-box-text" href="#">
+            <h6>
+              INSIGHTS
+            </h6>
+            <h3 class="main-head-text">
+              Satoshi Nakaboto: 'Grayscale exceeds $5B in Bitcoin under managment'
+            </h3>
+            <span>
+              by Satoshi Nakaboto
+            </span>
+            <span>
+              53 minutes ago
+              <i class="fas fa-share-alt"></i>
+              9
+            </span>
+          </a>
+        </div>
+        <div class="right-box">
+          <a class="head-box-text" href="#">
+            <h6>
+              TECH
+            </h6>
+            <h3>
+              Satoshi Nakaboto: 'Grayscale exceeds $5B in Bitcoin under managment'
+            </h3>
+            <span>
+              by James Briggs
+            </span>
+            <span>
+              1 hour ago
+              <i class="fas fa-share-alt"></i>
+              0
+            </span>
+          </a>
+        </div>
+        <div class="right-box-2">
+          <a class="head-box-text" href="#">
+            <h3>
+              Satoshi Nakaboto: 'Grayscale exceeds $5B in Bitcoin under managment'
+            </h3>
+            <span>
+              by Ivan Mehta
+            </span>
+            <span>
+              2 hours ago
+              <i class="fas fa-share-alt"></i>
+              12
+            </span>
+          </a>
+        </div>
+      </div>
+
+    </header>
+
+    <!-- END of header section -->
+
+    <!-- START of the latest news section -->
+
+    <main>
+      <section class="flex top-articles">
+        <div>
+          <h2>
+            <a href="#">
+              Latest news >
+            </a>
+          </h2>
+        </div>
+        <div class="container">
+          <article class="flex-box flex">
+            <figure>
+              <div class="article-img flex">
+                <img src="../src/assets/whats-app.png" alt="picture of whats-app">
+              </div>
+              <figcaption class="flex">
```

Comment:
#### Medium screen size

- [ ] It seems you are missing the breakpoints for the medium screen. CHeck at 1000px.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89058785-57071c80-d336-11ea-8ec2-e279f3c99596.png) |![image](https://user-images.githubusercontent.com/61479648/89058819-61c1b180-d336-11ea-8c17-3c215dd2a8b3.png) |

#### Small screen size

- [ ] Kindly fix the breakpoints

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89058955-a9e0d400-d336-11ea-901b-06bc6d3f10b8.png) | ![image](https://user-images.githubusercontent.com/61479648/89058976-b2390f00-d336-11ea-9a8b-7d69ff9872b4.png) |